core/config.js

```diff
@@ -20,7 +25,17 @@ module.exports = class Config {
       this.reset()
     }
 
-    this.config = require(this.configPath)
+    try {
+      this.config = require(this.configPath)
+    } catch (e) {
+      if (e instanceof SyntaxError) {
+        log.error(`Could not read config file at ${this.configPath}:`, e)
```

Comment:
Would it makes sense to attempt to remove the broken char before giving up? Could be in a separate PR.

Reply:
After discussing it, we decided not to make any attempt at fixing the config file.
:ballot_box_with_check: